doc/source/parameters/cfd/post_processing.rst

```diff
@@ -67,6 +67,8 @@ This subsection controls the post-processing other than the forces and torque on
     # Other Cahn-Hilliard postprocessing
     set calculate phase statistics       = false
     set phase statistics name            = phase_statistics
+    set calculate phase energy           = true
```

Comment:
True by default? It should be false, no?

Reply:
What do you think about removing the word "calculate" from the parameters? (can be another pr)

Reply:
should definitely be false, you're right. For the calculate I think I'll leave it to another PR